0_portfolio_input_check_functions.R

```diff
@@ -706,10 +704,9 @@ create_ald_flag <- function(portfolio, comp_fin_data, debt_fin_data){
   if (data_check(portfolio_other)){
     portfolio_other <- portfolio_other %>% mutate(has_asset_level_data = NA,
                                                   sectors_with_assets = NA,
-                                                  has_ald_in_fin_sector = NA,
-                                                  bics_sector = NA)
+                                                  has_ald_in_fin_sector = NA)
   }else{
-    portfolio_other <- portfolio_other %>% add_column("has_asset_level_data","sectors_with_assets","has_ald_in_fin_sector", "bics_sector")
+    portfolio_other <- portfolio_other %>% add_column("has_asset_level_data","sectors_with_assets","has_ald_in_fin_sector")
```

Comment:
I like the simplicity of this function! Can't get clearer as to intention on this!